app/views/admin/_service.erb

```diff
@@ -27,7 +27,10 @@
         <td class="govuk-table__cell"><%= service.updated_at %></td>
       </tr>
       <tr class="govuk-table__row">
-        <td class="govuk-table__cell govuk-table__cell--numeric" colspan="2"><%= link_to 'Manage', '', class: 'govuk-button', data: { module: 'govuk-button' } %></td>
+        <td class="govuk-table__cell govuk-table__cell--numeric" colspan="2">
+        <%= link_to 'Manage', '', class: 'govuk-button', data: { module: 'govuk-button' } %>
+        <%= link_to 'Delete', service_path(service), method: :delete, data: { confirm: 'Are you sure?' }, class: 'govuk-button govuk-button--warning' %>
```

Comment:
'Delete' could be localised... also 'Manage' from above has slipped through from a previous PR

Reply:
Didn’t think it was important to local the admin view see jakub https://github.com/alphagov/verify-self-service/pull/130 that’s why i didn’t localise 254

Reply:
👍